src/lang/builtins_string.ml

```diff
@@ -29,6 +29,15 @@ let _ =
       let s2 = Lang.to_string (Lang.assoc "" 2 p) in
       Lang.string (s1 ^ s2))
 
+let _ =
+  Lang.add_builtin ~base:string "digest" ~category:`String
+    ~descr:"Return an MD5 digest for the given string."
+    [("", Lang.string_t, None, None)]
+    Lang.string_t
+    (fun p ->
+      let data = Lang.to_string (List.assoc "" p) in
+      Lang.string Digest.(to_hex (string data)))
+
```

Comment:
Why is this change related to the cover?

Reply:
Not at all. I tried something, realized we didn't have `string.digest` and the reverted the changes but kept the new function.

I was trying to store the cover with a digest of its content but then realized we also need to remove them so I reverted..

Reply:
I've been testing it and reviewed your remark about the temp file, you are correct, the file should always be either the default cover or a unique temp file.

Reply:
I'll add this directly to main and will rebase here.